src/app/page.tsx

```diff
@@ -1,19 +1,17 @@
-'use client';
+// "use client";
```

Comment:
I believe you need this. If not, please remove

Reply:
I uncommented the line and it should work now.